README.md

```diff
@@ -51,6 +51,7 @@ None.
 | archive_src | A URL or a file path on the remote host pointing to an archive (tar or zip) containing the tool.  If left undefined then no archive will be installed, but the install directory will still be created and language-specific tooling will still be installed. | n/a | No |
 | csharp | A Boolean indicating whether or not the tool is written in C#; if it is then we will install the mono C# toolchain. | `false` | No |
 | go | A Boolean indicating whether or not the tool is written in Go; if it is then we will install the Go development toolchain. | `false` | No |
+| go_build | A Boolean indicating whether or not the Go tool should be built; if so then we will run `go build` from the project's root directory. | `true` | No |
```

Comment:
Why doesn't `go_build` default to `false`?

Reply:
Normally you want to build the Go tool, but I wanted to leave an escape hatch for tools that don't do things in the standard way; this is quite common with these assessment tools.

Reply:
Got it, thank you.